cmd/kms_priv_key_importer/readme.md

```diff
@@ -0,0 +1,113 @@
+ISSUER_VAULT_USERPASS_AUTH_ENABLED=false
+ISSUER_VAULT_USERPASS_AUTH_PASSWORD=issuernodepwd
+```
+
+Instead of setting the environment variables you use the `.env-issuer` file, you can copy the `.env-issuer.example` 
+file and rename it to `.env-issuer` and set the values of the environment variables there.
+Then from the root project folder you can run the following command **(just for vault o localstorage)**:
+
+```shell
+$ go run cmd/kms_priv_key_importer/main.go --privateKey <privateETHKey>
+````
+or you can build the binary and run it:
+
+```shell
+$ go build -o kms_priv_key_importer cmd/kms_priv_key_importer/main.go
+```
+
+and then run:
+
+```shell
+ ./kms_priv_key_importer --privateKey <privateETHKey>
+```
+
+
+### How import your private key to AWS KMS
+
+First you need to create a new key in AWS KMS, so export the variables defined in the requirements section and run the following command:
+
+```shell
+$ go run . --privateKey <privateETHKey>
+```
+To get the key id you have to take a look at the output (or logs) of the previous command, it will be something like:
+
+```logs
+2024/07/10 10:38:14 INFO alias created: alias:=alias/pbkey
+2024/07/10 10:38:14 INFO key created keyId=157a8b2a-e5e9-4414-b9c5-301ce828f6c5
+```
+
+then you can import your private key using the following command:
+
+```shell
+$ chmod +x kms_priv_key_importer
+$ ./kms_priv_key_importer <privateETHKey> <key-id> <aws-profile>
+```
+
+where:
+* `privateETHKey` is your private key in hex format (`ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`)
+* `key-id` is the key id of the key created in AWS KMS (in this example `157a8b2a-e5e9-4414-b9c5-301ce828f6c5`)
+* `aws-profile` is the profile name in your `~/.aws/credentials` file
+* `aws-region` is the region where the key was created
+
+if you get `Key material successfully imported!!!` message, then your private key was successfully imported to AWS KMS.
+
+### Running Importer with Docker (AWS)
+In the root project folder run:
+
+```shell
+docker build --build-arg ISSUER_KMS_ETH_PLUGIN_AWS_ACCESS_KEY=XXXX \
+  --build-arg ISSUER_KMS_ETH_PLUGIN_AWS_SECRET_KEY=YYYY \
+  --build-arg ISSUER_KMS_ETH_PLUGIN_AWS_REGION=eu-west-1 -t privadoid-kms-importer -f Dockerfile-kms-importer .
+```
+
+after the docker image is created run the following command (make sure you have the .env-issuer with your env vars):
+
+```shell
+docker run -it -v ./.env-issuer:/.env-issuer privadoid-kms-importer sh
+```
+
+inside the container `privadoid-kms-importer` execute:
+
+```
+./kms_priv_key_importer --privateKey <ETH-PRIV-KEY>
+```
+
+you will see something like this:
+
+```shell
+2024/07/10 15:27:54 INFO alias created: alias:=alias/pbkey
+2024/07/10 15:27:54 INFO key created keyId=9bb5b78b-c288-44a7-b1d4-0543e0a6
+```
+
+then import the material key
+
+```shell
+sh ./aws_kms_material_key_importer.sh ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 9bb5b78b-c288-44a7-b1d4-0543e0a6 privadoid
```

Comment:
`ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80` is a real private key with some balance. Please creat an empty one.